homework-submission/week1/app.js

```diff
@@ -0,0 +1,109 @@
+//Create a todo list called Static List with three list items:
+
+const staticList = [
+  { description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+  { description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+  { description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+];
+
+//1- Divide the list into reusable components (create at least 2):
+
+// A- reusable list title component:
+//the goal of this component is to create a reusable list title by using props
+class ListTitle extends React.Component {
+  render() {
+    const { title } = this.props;
+    return <h1>{title}</h1>;
+  }
+}
+
+// B- reusable li component
+//the goal of this component is to create a list item
+class Item extends React.Component {
+  render() {
+    // 4- Each list item should have a prop of description and deadline
+    const { description, deadline, done } = this.props;
+    return (
+      //6 Using CSS (a conditional className), cross out the item if done is true
+      <li className={done ? 'item-list' : ''}>
+        {description}, {deadline}
+      </li>
+    );
+  }
+}
+
+// 3- Reuse one component at least 3 times
+// the goal of this component is to make static component which make render to the title list and three items
+class List extends React.Component {
+  render() {
+    return (
+      <div>
+        <ListTitle title="Static List" />
+        <ul>
+          <Item description={staticList[0].description} deadline={staticList[0].deadline} />
```

Comment:
If you use an array to define the static list it's better to loop over it in my opinion. The alternative is writing the data here statically/hard-coded.

Reply:
I agree 👍 , I understood that we are required to use the Item component three times. So that's why I have it here like this while in the dynamic list I'm looping over the items.

Reply:
Ha I see what you mean :-)